contrib/registry/data/types/test/test-instance.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
```

Comment:
Github won't let me comment on the *.pyc files ... an we remove them from the repo, please?